hack/generator/pkg/conversions/property_assignment_function.go

```diff
@@ -90,6 +92,8 @@ func NewPropertyAssignmentToFunction(
 		knownLocals:     astmodel.NewKnownLocalsSet(idFactory),
 	}
 
+	result.knownLocals.Add("destination")
```

Comment:
Maybe add a comment that I (@unrepentantgeek) intend to do something that's less hacky here.